README.md

```diff
@@ -21,18 +21,27 @@ remotes::install_github("pharmaverse/admiral.test", ref = "devel")
+-   When you have created a program in the `dev/` folder, you need to run it as a standalone R script, in order to generate a test dataset that will become part of the {admiral.test} package, but you do not need to build the package.
+-   The programs in `dev/` are stored within the {admiral.test} GitHub repository, but they are **not** part of the {admiral.test} package--the `dev/` folder is specified in .Rbuildignore.
+-   When you run a program that is in the `dev/` folder, you generate a dataset that is written to the `data/` folder, which will become part of the {admiral.test} package.
+-   The names of test datasets are specified in `R/data.R`, for the purpose of generating documentation in the `man/` folder.
 
 ## Adding New SDTM Datasets
 
-- Add the output dataset name in `R/data.R` in the form `admiral_<name>`.
-- Create a program at `inst/data_scripts` named `<name>.R` (e.g. `rs.R`) to generate the test data and output as `admiral_<name>`. Use CDISC pilot data such as `admiral_dm` as input in this program in order to create realistic synthetic data that remains consistent with other domains. Note that **no personal data should be used** as part of this package, even if anonymized.
-- Add the output dataset to `data` folder
-- Run `devtools::document()` and include the updated `NAMESPACE` and `.Rd` files in `man/`
+-   Create a program in the `dev/` folder, named `<name>.R`, where `<name>` is the SDTM domain name, (e.g. `rs.R`), to generate the test data and output `admiral_<name>.rda` to the `data/` folder. Use CDISC pilot data such as `admiral_dm` as input in this program in order to create realistic synthetic data that remains consistent with other domains. Note that **no personal data should be used** as part of this package, even if anonymized.
+-   Run the program.
+-   Reflect this update, by specifying `admiral_<name>` in `R/data.R`.
+-   Run `devtools::document()` in order to update `NAMESPACE` and update the `.Rd` files in `man/`.
 
 ## Updating Existing SDTM Datasets
 
-- Rename the source dataset as `raw_<name>` (e.g. `raw_ds`) and reflect this change in `R/data.R` by including both `raw_<name>` and `admiral_<name>` here.
-- Create a program at `inst/data_scripts` named `update_<name>.R` to read in `raw_<name>`, make the updates, and output as `admiral_<name>`.
-- Add the output dataset to `data` folder
-- Run `devtools::document()` and include the updated `NAMESPACE` and `.Rd` files in `man/`
+-   Rename the source dataset as `raw_<name>.rda`, where `<name>` is the SDTM domain name, (e.g. `raw_ds.rda`), and copy it to the `data/` folder.
```

Comment:
Just renaming the file does not work. If you rename `admiral_dm.rda` to `raw_dm.rda` and call `data("raw_dm")` you still get `admiral_dm`. You have to recreate the `.rda` file with the new dataset name.

Reply:
@bundfussr , I've clarified by adding a bullet point to the general instructions and I've amended this line of text.